pavelib/servers.py

```diff
@@ -48,40 +54,63 @@ def run_server(system, settings=None, port=None, skip_assets=False, contracts=Fa
 @needs('pavelib.prereqs.install_prereqs')
 @cmdopts([
     ("settings=", "s", "Django settings"),
+    ("asset-settings=", "a", "Settings file used for updating assets. Defaults to settings if not provided."),
```

Comment:
"Defaults to settings if not provided." But only if settings aren't set... same note as above.